jekyll_get.gemspec

```diff
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = 'jekyll_get'
```

Comment:
I'm suggesting renaming the main module of this gem to `JekyllGet` (was `Jekyll_Get`). Classes and modules in Ruby use MixedCase and have no underscores. Naming the gem `jekyll_get` was a result of this decision, per Rubygems naming recommendations: http://guides.rubygems.org/name-your-gem/